data/mods/ssb/random-teams.js

```diff
@@ -790,6 +796,7 @@ class RandomStaffBrosTeams extends RandomTeams {
 			set.moves.push(ssbSet.signatureMove);
 			if (name === 'The Immortal' && set.item === 'Choice Scarf') set.moves[3] = 'Superpower';
 			if (name === 'irritated' && !set.moves.includes('Double Iron Bash')) set.moves[this.random(3)] = 'Double Iron Bash';
+			if (name === 'Rage') set.happiness = 0;
```

Comment:
Instead of this, support a happiness value in the SSBSet type.

Reply:
I did try doing that before but I couldn't get it to work

Reply:
@property {number=} happiness